mslib/msui/mscolab.py

```diff
@@ -260,6 +260,9 @@ def connect_handler(self):
                     self.loginButton.setEnabled(True)
             else:
                 show_popup(self, "Error", "Some unexpected error occurred. Please try again.")
+        except requests.exceptions.SSLError:
```

Comment:
I noted some time ago tat I got the wrong ConnectionError and noted this hear
https://github.com/Open-MSS/MSS/issues/673#issuecomment-830837312

Moving the SSL on top is a good idea and helps to find faster when you have e.g. an incomplete chain certificate.

But what is with the others? If they also get catched by the ConnectionError, we better use the chance now to move that on the end of the list.

Reply:
As far as I can see ConnectionError is only the superclass of SSLError. The others get caught correctly.
While not specifically tested, I did have a look at the mock output and made sure this was the case.

Should I move it to the bottom regardless, for the future?

Reply:
I moved it to the bottom now. In case we need more errors in the future this is safer.

Reply:
Thx